### DISCUSSION TITLE
ive just created the korean language pack

### DISCUSSION THREAD

**Discussion by randomizermk1** [monkeytypegame/monkeytype#810] [Ideas] [answered]
the thing is, i'm not quite familiar with git. should I push or pull? commit? if then to what branch??? what am I suppose to do now? I also want to create a short quote pack for korean. how am I suppose to do that? halp plz ....

**Comment by Miodec** [comment, marked as answer]
Korean won't be fully supported because of the core method of input handling im afraid. It needs a rewrite.

**Reply by jeremy-beasley** [reply]
@Miodec was going to create a FR but saw this first while searching for `Korean`. Is this **_input handler_** still the blocker? Will this continue to be a blocker for the foreseeable future?

**Reply by Miodec** [reply]
Yeah, still an issue. Haven't had time to even have a look at Korean accuracy calcuation.

**Reply by jeremy-beasley** [reply]
Got it. I may start to sniff around. Did you have any resources bookmarked for Korean or any languages that exhibit similar properties?
